### PR TITLE
[4.4] Remove duplicate png

### PR DESCRIPTION
**Pull request by brianteeman**
simple pr to correct an error from #34634 where png was added twice

Pull Request for Issue #41898  .

### Link to documentations
Please select:
- [ ] Documentation link for docs.joomla.org: <link>
- [ ] No documentation changes for docs.joomla.org needed

- [ ] Pull Request link for manual.joomla.org: <link>
- [ ] No documentation changes for manual.joomla.org needed

### CODE REVIEW

**Comment by joomleb** [comment]
Tested **without success**
I used the update zip package and I tried also the new installation full zip package

**Comment by brianteeman** [comment]
@joomleb sorry I should have said that this is only for new installs.

**Comment by sandewt** [comment]
https://github.com/brianteeman/joomla-cms/blob/b6781aa2df4c84ed99dad8d76b3fc4ba78a62f9a/installation/sql/mysql/base.sql#L168

png was added twice

**Comment by brianteeman** [comment]
oops - thanks sql updated as well

**Comment by joomleb** [comment]
@brianteeman 
Please, Can you confirm that package has been updated and I can re-test it ?

**Comment by brianteeman** [comment]
yes - the build date/time is on the download page

**Review by joomleb — APPROVED** [review]

File: administrator/components/com_media/config.xml [hunk 1]
```diff
@@ -55,7 +55,7 @@
 			type="textarea"
 			label="COM_MEDIA_FIELD_RESTRICT_UPLOADS_EXTENSIONS_LABEL"
 			description="COM_MEDIA_FIELD_RESTRICT_UPLOADS_EXTENSIONS_DESC"
-			default="bmp,gif,jpg,jpeg,png,webp,ico,mp3,mp4,odg,odp,ods,odt,pdf,png,ppt,txt,xcf,xls,csv"
+			default="bmp,gif,jpg,jpeg,png,webp,ico,mp3,mp4,odg,odp,ods,odt,pdf,ppt,txt,xcf,xls,csv"
```
Comment (Quy) [inline administrator/components/com_media/config.xml:58]: These should match with the SQL file. Do in a separate PR?

**Comment by HLeithner** [comment]
This pull request has been automatically rebased to 4.4-dev.

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on 1e7a64cdfbcb51e81e9ab17326d7e9d5cf24265d<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/41900">issues.joomla.org/tracker/joomla-cms/41900</a>.</sub>

**Comment by alikon** [comment]
I have tested this item :white_check_mark: successfully on 1e7a64cdfbcb51e81e9ab17326d7e9d5cf24265d<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/41900">issues.joomla.org/tracker/joomla-cms/41900</a>.</sub>

**Comment by alikon** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/41900">issues.joomla.org/tracker/joomla-cms/41900</a>.</sub>

**Comment by joomleb** [comment]
Hi guys,
Why this PR has not been merged into 4.4 ?
Please, What is missing?

**Comment by laoneo** [comment]
Thanks!
